about.html

```diff
@@ -0,0 +1,154 @@
+                  <a href="./index.html">Home</a>
+                </li>
+                <li class="desk-navbar-menu-item">
+                  <a href="./about.html">About</a>
+                </li>
+                <li class="desk-navbar-menu-item">
+                  <a href="#">Projects</a>
+                </li>
+                <li class="desk-navbar-menu-item">
+                  <a href="#">Contact</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <ul class="menu-list">
+            <li class="navbar-menu-item">
+              <a href="./index.html">Home</a>
+            </li>
+            <li class="navbar-menu-item">
+              <a href="./about.html">About</a>
+            </li>
+            <li class="navbar-menu-item">
+              <a href="#">Projects</a>
+            </li>
+            <li class="navbar-menu-item">
+              <a href="#">Contact</a>
+            </li>
+          </ul>
+          <span class="campaign">
+            <a href="#"> CC Campaign </a>
+          </span>
+        </nav>
+      </section>
+
+      <section class="banner">
+        <h1>ARTWORK since many years</h1>
+                <p class="banner-description">This art exhibition has played a crucial part in the market for new art since the 18th and 19th centuries.
+                        The Paris Salon, open to the public from 1737, rapidly became the key factor in determining the reputation, 
+                        and so the price, of the French artists of the day. </p>
+       
+      </section>
+    </header>
+
+   
+    <div class="container section-product">
+      <h3 class="d-flex justify-content-center card-det">Artworks events - 2022
+         </h3>
+      <div class="container d-flex my-3 justify-content-center line-height"></div>
+      <p class="container text-center d-flex justify-content-center text-card">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+          culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+          iusto at?
+      </p>
+      <div class="logo-about d-flex justify-content-center"><img src="./img/artevent.png" alt="logo"></div>
+  </div>
+  <div class="pt-5 rank-section-about">
+      <h3 class="d-flex justify-content-center title-rank">We started in 2013</h3>
+      <div class="container d-flex my-3 justify-content-center line-height"></div>
+      <p class="container d-flex justify-content-center text-center rank-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+          culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+          iusto at?
+      </p>
+
+      <div class="production-container container-fluid testymonies mb-5 testymonies2">
+      </div>
+  </div>
+  <main>
+      <section class="partners">
```

Comment:
- Optional
As per the Behance design template, this section is not included in the original design's desktop mode. It would be better if you would exclude this part from the about page in the desktop mode